server/cmd/oxidrive/main.go

```diff
@@ -20,7 +21,7 @@ import (
 func main() {
 	trapSigterm()
 
-	cfg := ParseConfig()
+	cfg := config.ParseConfig()
```

Comment:
```suggestion
	cfg := config.Parse()
```

Less redundant maybe?

Reply:
That's idiomatic Go, good one